.ado/templates/react-native-macos-init.yml

```diff
@@ -81,15 +81,10 @@ steps:
       npx beachball publish --branch origin/$(System.PullRequest.TargetBranch) --no-push --registry http://localhost:4873 --yes --access public
     displayName: Publish react-native-macos-init to verdaccio
 
-  - task: CmdLine@2
-    displayName: Install react-native cli
-    inputs:
-      script: npm install -g react-native-cli
-
   - task: CmdLine@2
     displayName: Init new project
     inputs:
-      script: npx react-native init testcli
+      script: npx --yes react-native@0.68.2 init testcli --template react-native@0.68.2
```

Comment:
Will we have to update this with every version bump?

Reply:
should only need it on major version bumps.  I can file an issue to make this logic something more robust.